<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuForm</class>
 <widget class="QWidget" name="MenuForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>38</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="campaign">
            <property name="text">
             <string>Кампания</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="continueGame">
            <property name="text">
             <string>Продолжить игру</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gameWithSetting">
            <property name="text">
             <string>Своя игра</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exit">
            <property name="text">
             <string>Выход</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>campaign</sender>
   <signal>clicked()</signal>
   <receiver>MenuForm</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gameWithSetting</sender>
   <signal>clicked()</signal>
   <receiver>MenuForm</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
